nautobot/docs/development/apps/api/setup.md

```diff
@@ -53,9 +53,7 @@ The top level is the project root. Immediately within the root should exist seve
 The app source directory contains all of the actual Python code and other resources used by your app. Its structure is left to the author's discretion, however it is recommended to follow best practices as outlined in the [Django documentation](https://docs.djangoproject.com/en/stable/intro/reusable-apps/). At a minimum, this directory **must** contain an `__init__.py` file containing an instance of Nautobot's `NautobotAppConfig` class.
 
 !!! note
-    Nautobot includes a command to help create the app directory:
-    `nautobot-server startplugin [app_name]`
-    Please see the [Nautobot Server Guide](../../../user-guide/administration/tools/nautobot-server.md#startplugin) for more information.
+    The Nautobot organization provides a Python [CookieCutter](https://cookiecutter.readthedocs.io/en/stable/) to help get started with your applications. Get started at [https://github.com/nautobot/cookiecutter-nautobot-app](https://github.com/nautobot/cookiecutter-nautobot-app).
```

Comment:
Probably would be good to add a `changes/5078.documentation` with something like:

```
Added a link to the `cookiecutter-nautobot-app` project in the App developer documentation.
```

Reply:
I'm starting to get the picture a bit more on how Town Crier works with this. Thank you for the patience.